libs/components/toast/src/lib/modules/toast/toast.service.ts

```diff
@@ -26,21 +26,22 @@ import { SkyToastConfig } from './types/toast-config';
 })
 export class SkyToastService implements OnDestroy {
   private static host: ComponentRef<SkyToasterComponent> | undefined;
+  private static toasts: SkyToast[] = [];
+  private static toastStream: BehaviorSubject<SkyToast[]>;
```

Comment:
`toasts` and `toastStream` need to be static so that both `SkyToastService` and `SkyToastLegacyService` contribute to the same pool of toast instances.